beta-src/src/components/ui/WDInfoPanel.tsx

```diff
@@ -34,8 +42,19 @@ const WDInfoPanel: React.FC<WDInfoPanelProps> = function ({
       ...voteState,
       [voteKey]: !voteState[voteKey],
     };
+    const urlParams = new URLSearchParams(window.location.search);
+    const currentGameID = urlParams.get("gameID");
+    const countryID = userCountry.countryID.toString();
```

Comment:
shouldn't we be getting this from state instead of the url?

Reply:
This is how it was pulled from App so I just followed that. It starts acting weird with the types when I try to pull it straight from state

Reply:
In App.tsx we have to load currentGameID from the url from because this is the initial load. Once WDUI is mounted we should have access to state.